pkgs/development/python-modules/piccata/default.nix

```
@@ -1,16 +1,21 @@
-{ buildPythonPackage, fetchPypi, lib, ipaddress, isPy3k }:
+{ buildPythonPackage, fetchFromGitHub, lib, ipaddress }:
 
 buildPythonPackage rec {
   pname = "piccata";
-  version = "1.0.1";
-  disabled = isPy3k;
+  version = "2.0.0";
```

actually, reading through the failures mores:
```suggestion
  version = "2.0.0";
  disabled = isPy27;
```

```
ilder for '/nix/store/1s8wmh5afdh89m3r6lxyy92jvi75xs6x-python2.7-piccata-2.0.0.drv' failed with exit code 1; last 10 log lines:
      from abc import ABC, abstractmethod
  ImportError: cannot import name ABC


  ----------------------------------------------------------------------
  Ran 1 test in 0.000s

  FAILED (errors=1)
  Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
```

---

Thank you, much appreciated